chrome/content/zotero/preferences/preferences_general.xul

```diff
@@ -58,6 +59,17 @@
 				<rows id="zotero-prefpane-general-rows">
 					<row>
 						<hbox align="center">
+							<label value="&zotero.preferences.layout;" control="layout"/>
+						</hbox>
+						<radiogroup id="layout" orient="horizontal" align="center" preference="pref-layout"
+							onblur="Zotero.Prefs.set('layout', this.value);">
```

Comment:
Y'know, I could have sworn I had trouble with getting this to work right otherwise, but now that I take it out again it works. Guess I was too hasty the first time through.